code/modules/ordeals/gold_ordeals.dm

```diff
@@ -0,0 +1,79 @@
+// Dawn
+/datum/ordeal/gold_dawn
```

Comment:
Subtype off SimpleCommander

Reply:
The code isn't quite robust enough to do that. I need the roamers to share a list of spawn locations with the simplecommander groups, and that can't really be done through subtyping.